package.json

```diff
@@ -41,7 +41,11 @@
         "doc": "docs"
     },
     "devDependencies": {
-        "@pxblue/prettier-config": "^1.0.2",
+        "@pxblue/prettier-config": "^1.0.3",
         "prettier": "^2.2.1"
+    },
+    "dependencies": {
+        "@pxblue/eslint-config": "^2.0.5",
+        "@pxblue/ng-progress-icons": "^1.2.1"
     }
```

Comment:
ESLint should be in devDependencies. 
Why are we adding progress icons dependency here?